Xamarin.Forms.Core/Image.cs

```diff
@@ -21,6 +21,17 @@ public class Image : View, IImageController, IElementConfiguration<Image>
 
 		public static readonly BindableProperty IsLoadingProperty = IsLoadingPropertyKey.BindableProperty;
 
+		public enum AnimationPlayBehaviorValue
+		{
+			None,
+			OnLoad,
+			OnStart
+		};
+
+		public static readonly BindableProperty AnimationPlayBehaviorProperty = BindableProperty.Create(nameof(AnimationPlayBehavior), typeof(AnimationPlayBehaviorValue), typeof(Image), AnimationPlayBehaviorValue.None);
+
+		public static readonly BindableProperty IsAnimationPlayingProperty = BindableProperty.Create (nameof(IsAnimationPlaying), typeof (bool), typeof (Image), false);
```

Comment:
I guess you want a read-only property ? not exposing a setter isn't enough. As someone could still set the value from XAML, from a Binding, or by using `SetValue`.

what you want instead is like this:
```
static readonly BindablePropertyKey IsAnimationPlayingPropertyKey = BindableProperty.CreateReadOnly (nameof(IsAnimationPlaying), typeof (bool), typeof (Image), false);
public static readonly BindableProperty IsAnimationPlayingProperty = IsAnimationPlayingPropertyKey.BindableProperty;
```

you can even expose a private setter

```
public bool IsAnimationPlaying
{
  get { return (bool)GetValue(IsAnimationPlayingProperty); }
  private set => SetValue(IsAnimationPlayingPropertyKey, value);
}
```

Reply:
Fixed.